src/strokes.ts

```diff
@@ -243,8 +233,14 @@ export class Stroke extends Modifier {
 
     // Draw the rasquedo "R"
     if (this.type === Stroke.Type.RASQUEDO_DOWN || this.type === Stroke.Type.RASQUEDO_UP) {
+      const textFont = {
+        family: Tables.lookupMetric(`Strokes.text.fontFamily`),
+        size: Tables.lookupMetric(`Strokes.text.fontSize`),
+        weight: Tables.lookupMetric(`Strokes.text.fontWeight`),
+        style: Tables.lookupMetric(`Strokes.text.fontStyle`),
+      };
```

Comment:
Is this because Strokes will have a different fontFamily than Strokes.text?

For example, Strokes will use `Bravura` while Strokes.text will use `12pt Arial`?

Reply:
yes :)